ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md

```diff
@@ -375,6 +375,12 @@ bfd_multihop:
 # Requires use eBGP as overlay protocol.
 evpn_overlay_bgp_rtc: < true | false , default -> false >
 
+# Configure route-map on eBGP sessions towards route-servers, where the peer's ASN is filtered away.
```

Comment:
Configure route-map on eBGP sessions towards route-servers, where we match prefixes based on the peer's ASN to be filtered away.